src/main/java/org/broadinstitute/consent/http/models/elastic_search/DatasetTerm.java

```diff
@@ -0,0 +1,170 @@
+package org.broadinstitute.consent.http.models.elastic_search;
+
+import java.util.List;
+import org.broadinstitute.consent.http.models.ontology.DataUseSummary;
+
+public class DatasetTerm {
```

Comment:
Hmm, I do agree, but in trying it out, the sheer number of properties makes it hard to work with, since records are immutable and have to be provided all the properties at once. Building it up one by one I think is a lot cleaner. If you don't mind a bit of clunkiness, I am happy to change it, though

Reply:
I'm fine with this approach if that's what we're doing.